node/node_syncing.go

```diff
@@ -230,7 +230,10 @@ SyncingLoop:
 			if willJoinConsensus {
 				node.Consensus.BlocksNotSynchronized()
 			}
-			node.stateSync.SyncLoop(bc, worker, willJoinConsensus, false)
+			node.stateSync.SyncLoop(bc, worker, false)
+			if node.NodeConfig.Role() == nodeconfig.ExplorerNode {
```

Comment:
NodeConfig.Role() == nodeconfig.ExplorerNode means willJoinConsensus == false means node. Currently, node being explorer is the only case willJoinConsensus == false.

It would be better if we use 'if (will_join_consensus}) { .sync state } else { .updateConsensusInformation }

Reply:
@chaosma @rlan35 could you correct me if there is misunderstanding on what I stated above? :)